src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -87,6 +87,9 @@ function transform(node) {
 		extractedAttributes[attributeName] = attributeValue;
 	}
 	
+	if (node.checked) {
```

Comment:
Shouldn't we treat it like the `disabled` attribute? `disabled` is handled via its own method `isDisabled(node)` and will always be `true` or `false`, we just treat `false` as a default so that it's not part of the resulting state. In contrast, checked is `true` or not present at all?

I would suggest to do the same here: Have a method like `isChecked(node)` that always returns `true` or `false` like:

```js
extractedAttributes["checked"] = isChecked(node);
```

Then add `checked: false` to our `defaults.yaml`.

This way, we are not just consistent when it comes to handling possibly absent attributes, the user can configure the behavior via our defaults mechanism.

Reply:
Hmm. The difference is, that `disabled` is a property of all `input` elements, whereas `checked` is a property only of `input` elements of type `checkbox`. But I do see your point, and we can change that accordingly...

Reply:
I think the idea is not to check for `if(node.checked)`, so that `"checked"` has always a value with either `true` or `false` and cannot contain three states with `undefined`.
If I understand @beatngu13 correctly, he suggest to do something like (via JavaScript-Magic):
```suggestion
	extractedAttributes["checked"] = node.checked || false;
```

Reply:
As long as it is hidden in separate method (like `isDisabled`, i.e., `isChecked`) and the user can leverage our defaults-mechanism, I'm good. 😉